packages/graph/hash_graph/lib/graph/src/datastore/postgres/mod.rs

```diff
@@ -451,7 +454,20 @@ impl PostgresDatabase {
                 .change_context(InsertionError)?;
         }
 
-        // TODO: Store link references
+        // FIXME: Without `collect` we get a weird lifetime error
+        let entity_type_references = entity_type
+            .inner()
+            .link_references()
+            .into_iter()
+            .map(|(_link_uri, entity_type_uri)| entity_type_uri)
+            .collect::<Vec<_>>();
+
+        // TODO: Store references in the database
+        let _entity_type_reference_ids =
+            Self::entity_type_reference_ids(transaction, entity_type_references)
+                .await
+                .change_context(InsertionError)
+                .attach_printable("Could not find referenced link")?;
```

Comment:
Same as above

Reply:
Changed to `"Could not find referenced entity"` which matches the actual error: e79e050e8864bfe4f92bc2209115ed70d93499dd

Reply:
They are entity types, not entities 🙃